assets/js/modules/adsense/datastore/service.js

```diff
@@ -106,9 +106,10 @@ export const selectors = {
 	 *
 	 * @since 1.14.0
 	 *
+	 * @param {Object} urlParams URL parameters to be passed to the query.
 	 * @return {(string|undefined)} AdSense account site overview URL (or `undefined` if not loaded).
 	 */
-	getServiceAccountSiteURL: createRegistrySelector( ( select ) => () => {
+	getServiceAccountSiteURL: createRegistrySelector( ( select ) => ( state, urlParams = {} ) => {
```

Comment:
@aaemnnosttv noticed during testing that the URL returned by this selector is actually not the correct one to pass these date parameters too, which I just verified.

All these links instead need to go to a `https://www.google.com/adsense/new/u/0/{accountID}/reporting/` base URL. Let's introduce a simple AdSense `getServiceReportURL( reportArgs )` registry selector, similar to the one we have for Analytics. It should do the following:
* Get the `accountID`.
* Call the `getServiceURL( { path, query } )` selector with `path` set to `${ accountID }/reporting` and `query` set to the `reportArgs` provided, simply passing through.

Let's then revert the changes made to `getServiceAccountSiteURL` here and rely on the new selector throughout this PR. Its extra arguments like `source` and `url` are irrelevant for report URLs and thus shouldn't be provided with the new selector.